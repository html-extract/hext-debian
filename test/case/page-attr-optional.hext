<a href:link name:anchor? />
