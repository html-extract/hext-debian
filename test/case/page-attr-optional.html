<a href="link1" name="anchor1"></a>
<a href="link2"></a>
<a href="link3" name="anchor3"></a>
<a href="link4" name=""></a>
<a name="anchor5 no"></a>
