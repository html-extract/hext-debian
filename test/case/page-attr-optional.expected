{"anchor":"anchor1","link":"link1"}
{"link":"link2"}
{"anchor":"anchor3","link":"link3"}
{"anchor":"","link":"link4"}
